sql/src/test/resources/calcite/tests/window/WindowOpReorder.sqlTest

```diff
@@ -0,0 +1,51 @@
+type: "operatorValidation"
+
+sql: |
+    SELECT
+      m1,
+      m2,
+      SUM(m1) OVER(PARTITION BY m2) as sum1,
+      SUM(m2) OVER() as sum2
+    from druid.numfoo
+    GROUP BY m1,m2
+
+expectedOperators:
+  - type: "naivePartition"
+    partitionColumns: [ ]
```

Comment:
not necessarily right now - but I think a check should be added to the `NaivePartitioningOperator`:
* if the `partitioningColumns` is empty
* it may only get at most `1` RACs 
I wonder if we've already violated that or not - but probably not...